packages/flutter/lib/src/rendering/sliver_fill.dart

```diff
@@ -124,7 +134,7 @@ class RenderSliverFillRemaining extends RenderSliverSingleBoxAdapter {
     assert(paintedChildSize.isFinite);
     assert(paintedChildSize >= 0.0);
     geometry = SliverGeometry(
-      scrollExtent: constraints.viewportMainAxisExtent,
+      scrollExtent: scrollBody ? constraints.viewportMainAxisExtent : constraints.scrollOffset,
```

Comment:
is it if scrollBody is true, it will fill the children to a viewport extent, fill the remaining view port extend if false?

Reply:
If it is the case, I feel the name is a little bit confusing, maybe fillFullExtent or even create a new widget.